src/annotator/config/index.js

```diff
@@ -60,71 +68,100 @@ function configurationKeys(appContext) {
 }
 
 /**
- * Reads the Hypothesis configuration from the environment.
- *
- * @param {string[]} settingsKeys - List of settings that should be returned.
- * @param {Window} window_ - The Window object to read config from.
+ * The definition for configuration sources, their default values
```

Comment:
```suggestion
 * Definitions of configuration keys
```

The entries in this may are configuration _keys_ rather than _sources_ (which I take to mean JSON script tags, link elements etc.). I'm not completely sure about the term "definition" vs say "metadata", but lets roll with it.